app/models/order.rb

```diff
@@ -79,27 +80,17 @@ def set_default_address
     end
 
     def initialize_business_days
-      @min_day = business_days_after(BUSINESS_DAY_FROM)
-      @max_day = business_days_after(BUSINESS_DAY_TO)
-    end
-
-    def business_days_after num
-      date = Date.current
-
-      while num > 1
-        date += 1
-        num -= 1 if 1 <= date.wday and date.wday <= 5
-      end
-
-      date
+      @min_day = BUSINESS_DAY_FROM.business_days.from_now.to_date
+      @max_day = BUSINESS_DAY_TO.business_days.from_now.to_date
     end
 
     def delivery_day_limit
-      errors.add :delivery_day, :blank and return if delivery_day.blank?
-      if delivery_day < min_day or max_day < delivery_day or [0, 6].include?(delivery_day.wday)
-        errors.add(:delivery_day,
-          I18n.t('activerecord.errors.messages.invalid_business_day',
-            from: BUSINESS_DAY_FROM, to: BUSINESS_DAY_TO))
+      if delivery_day.present?
+        unless delivery_day.between?(min_day, max_day) && delivery_day.workday?
+          errors.add(:delivery_day,
+            I18n.t('activerecord.errors.messages.invalid_business_day',
```

Comment:
[rubocop] Align the parameters of a method call if they span more than one line.